browser/ui/views/brave_shields/cookie_list_opt_in_browsertest.cc

```diff
@@ -138,7 +153,7 @@ IN_PROC_BROWSER_TEST_F(CookieListOptInBrowserTest, EnableFromBubble) {
     })
   )js"));
 
-  run_loop.Run();
+  enabled_observer.Wait();
 
   EXPECT_TRUE(IsCookieListFilterEnabled());
```

Comment:
Given that `IsCookieListFilterEnabled()` is the escape condition of the loop, this is not necessary anymore, right?

Reply:
I noticed that as well - I think it's probably best to leave the `EXPECT` here, to be explicit and also to catch if something doesn't work quite right with the observer.